test/features/user_edits_a_robot_test.rb

```diff
@@ -0,0 +1,38 @@
+require_relative '../test_helper'
+
+class UserEditsARobotTest < FeatureTest
+  def test_user_can_edit_a_robot
+    robot_1 = {:name => 'Ryan',
+               :city => 'State College', :state => 'PA',
+               :avatar => 'https://robohash.org/249',
+               :birthdate => '05/31/1985', :date_hired => '05/09/2016',
+               :department => 'Web Dev'}
+    robot_2 = {:name => 'Rando',
```

Comment:
because you're creating these hash's of data in different tests, you could perhaps create a method in a helper module that returns this data for you so you don't have to re-write it!